hedyorg/hedy
# 🧹 Remove button from grammars and tranpiler

Fixes #5677
Removes `button` syntax from the grammars, the transpiler and the tests. There should be no references to _BUTTON, assign_button or create_button.

**How to test**
Navigate to level 5 and ensure that the following command does not yield the error `name 'create_button' is not defined`.
```
hello is button
if hello is pressed print "Hello world!"
else print 'not'
```
